docker-storage-setup-override.conf

```diff
@@ -1,4 +0,0 @@
-# Edit this file to override any configuration options specified in
-# /usr/lib/docker-storage-setup/docker-storage-setup.
-#
-# For more details refer to "man docker-storage-setup"
```

Comment:
Yes that is the idea.  But we would just ship /etc/sysconfig/docker-storage-setup and /etc/sysconfig/ocid-storage-setup. Perhaps with a /usr/share/ocid-storage-setup file available for users to play with.